src/main/java/xbot/common/controls/actuators/XCANSparkMax.java

```diff
@@ -19,24 +24,31 @@ public abstract class XCANSparkMax {
 
     protected int deviceId;
     protected String prefix = "";
+    protected DeviceInfo info;
     PropertyFactory propertyFactory;
 
-    final DoubleProperty kPprop;
-    final DoubleProperty kIprop;
-    final DoubleProperty kDprop;
-    final DoubleProperty kIzProp;
-    final DoubleProperty kFFprop;
-    final DoubleProperty kMaxOutputProp;
-    final DoubleProperty kMinOutoutProp;
+    private boolean usesPropertySystem = true;
```

Comment:
I've been thinking in general we probably should keep things protected rather than private so that extension is more possible?